### PR TITLE
COMMERCE-1738 added check on array emptiness

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

File: commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentMethodGroupRelsDisplayContext.java [hunk 1]
```diff
@@ -151,9 +151,11 @@ else if (navigation.equals("inactive")) {
 			results = addDefaultCommercePaymentMethodGroupRels(results);
 		}
 
-		results.sort(
-			new CommercePaymentMethodGroupRelNameComparator(
-				themeDisplay.getLocale()));
+		if (!results.isEmpty()) {
+			results.sort(
```
Comment (ethanbustad) [inline commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentMethodGroupRelsDisplayContext.java]: Hey @andrea-ale-sbarra, let's have a call about this tomorrow. I think this isn't the right fix.

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:tets

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Review by ethanbustad — CHANGES_REQUESTED** [review]
Nice, @andrea-ale-sbarra, the only problem I see is we need to assign the result of the `ListUtil.sort` call back to the `results` variable -- otherwise, _if_ it's an unmodifiable list, we won't have access to the sorted result. (If the list is modifiable, then obviously the sort will be done on the original list, so we'll have access to the sort in that case.)

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

**Pull request merged**